netgrasp/netgrasp.py

```diff
@@ -502,20 +503,21 @@ def ip_has_changed(did):
         else:
             ng.debugger.debug("ip for did %s has not changed", (did,))
             return False
-
     except Exception:
         ng.debugger.dump_exception("ip_has_changed() caught exception")
 
 
 # Database definitions.
 def create_database():
+    from .database import SCHEMA_VERSION_KEY
+    from .database import SCHEMA_VERSION
+
```

Comment:
I like these changes (moving these into defines in __init__), but the belong in a different PR -- this one is touching way too many unrelated things and needs to focus only on the local mac changes.